qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/CassandraMetricsClientLimitMultiplier.java

```diff
@@ -42,8 +42,8 @@ private CassandraMetricsClientLimitMultiplier(
 
     public static ClientLimitMultiplier create(Supplier<QosCassandraMetricsRuntimeConfig> runtimeConfig,
             QosCassandraMetricsInstallConfig installConfig, ScheduledExecutorService metricsLoaderExecutor) {
-        CassandraMetricsService metricsService = JaxRsClient.create(
-                CassandraMetricsService.class,
+        MetricsService metricsService = JaxRsClient.create(
```

Comment:
This would mean that we try to run a getMetric RPC on the cassandraService, which will fail with a `NotFoundException` as the Cassandra service doesnt have the `MetricsService` endpoint. We need to create the client internally to the actual CassandraMetricsService by taking the required internal dependency.